components/home/integrationDestinations.tsx

```diff
@@ -5,24 +5,28 @@ export default function IntegrationDestinations() {
     <>
```

Comment:
This can be determined programmatically now!   You can take the top N destinations by priority and display them